plugins/azure_sentinel/help.md

```diff
@@ -0,0 +1,613 @@
+# Description
+
+Azure Sentinel is Microsoft's' automated security service.
+
+# Key Features
+
+* Creating and updating incidents
+* Deleting incidents
+* Retriveing incident's details
+* Listing incidents for a given workspace
+* Listing bookmarks for a given incident
+* Listing alerts for a given incident
+
+# Requirements
+
+* Set of credentials with necessary permissions to monitor and modify Sentinel incidents
+
+# Supported Product Versions
+
+* 2021-04-01
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|client_id|string|None|True|The application ID that the application registration portal assigned to your app|None|c163eff0-d1a1-4618-ee2a-453534f43cee|
+|client_secret|credential_secret_key|None|True|The application secret that you generated for your app in the app registration portal|None|ef50c6bx9umaik9agvoxtoqec2fg9f0y|
+|tenant_id|string|None|True|This is active directory ID|None|5ceea899-ae8c-4ff1-fffe-353646eeeff0|
+
+Example input:
+
+```
+{
+  "client_id": "c163eff0-d1a1-4618-ee2a-453534f43cee",
+  "client_secret": "ef50c6bx9umaik9agvoxtoqec2fg9f0y",
+  "tenant_id": "5ceea899-ae8c-4ff1-fffe-353646eeeff0"
+}
+```
+
+## Technical Details
+
+### Actions
+
+#### List Entities
+
+This action is used to get all incidents entities.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|incidentId|string|None|True|Incident ID|None|incident123|
+|resourceGroupName|string|None|True|The name of the resource group within the user's subscription|None|resourcegrup12|
+|subscriptionId|string|None|True|Azure subscription ID|None|0caafeeb-aaa0-44ca-ffe1-aaaaeeeffffe|
+|workspaceName|string|None|True|The name of the workspace|None|workspace23|
+
+Example input:
+
+```
+{
+  "incidentId": "incident123",
+  "resourceGroupName": "resourcegrup12",
+  "subscriptionId": "0caafeeb-aaa0-44ca-ffe1-aaaaeeeffffe",
+  "workspaceName": "workspace23"
```

Comment:
Just wondering right now - should those parameters stay as inputs for action or maybe it would be much better for them to stay in connection?